<launch>
  <node pkg="nodelet" type="nodelet" name="nodelet_manager" args="manager" />

  <node pkg="nodelet" type="nodelet" name="nodelet1"
        args="load depth_image_proc/point_cloud_xyz nodelet_manager">
    <remap from="camera_info" to="/uav1/rs_d435/aligned_depth_to_color/camera_info"/>
    <remap from="image_rect" to="/uav1/rs_d435/aligned_depth_to_color/image_raw"/>
    <remap from="points" to="/uav1/rs_d435/depth/points"/>
  </node>
</launch>